Resources/Textures/_Harmony/Clothing/Uniforms/Jumpskirt/janitor.rsi/meta.json

```
@@ -0,0 +1,30 @@
+{
+  "version": 1,
+  "license": "CC-BY-SA-3.0",
```

Goon appears to say that all of their assets are CC-BY-NC-SA-3.0.
If your sprites are considered an update to the old sprites, they have to be shared under the same terms.
```suggestion
  "license": "CC-BY-NC-SA-3.0",
```